Assets/UI/Scripts/HelpUI.gd

```diff
@@ -0,0 +1,9 @@
+extends Control
+class_name HelpUI
+
+var parent = null
+
+func _on_Button_pressed():
+	Audio.play_snd_click()
+	parent.visible = true
```

Comment:
Invalid set index 'visible' (on base: 'Nil') with value of type 'bool'.

Reply:
Same problem with the commit. Should be fixed now.